README.md

```diff
@@ -78,6 +82,20 @@ found in the `src/` directory.
     - stylesheets/
 ```
 
+`require('uswds')` will load all of Draft U.S. Web Design Standard's JavaScript onto the page. The uswds module itself does not export anything.
+
+The main Sass (SCSS) source file is here: 
+
+```
+node_modules/uswds/src/stylesheets/all.scss
+```
+
+The non-minified CSS that’s been precompiled is here:
```

Comment:
Why not point them to the minified versions, as well as the minified/non-minified JS too?